stdlib/public/core/StringUTF8.swift

```diff
@@ -391,7 +391,10 @@ extension String {
   /// slice of the `picnicGuest.utf8` view.
   ///
   /// - Parameter utf8: A UTF-8 code sequence.
-  @available(swift, deprecated: 3.2, obsoleted: 4.0)
+  @available(swift, deprecated: 3.2, message:
```

Comment:
Just a stylistic nit, I'd move the `message:` indented one level to the next line. Since the message is already beyond 80 columns.

Reply:
What is the formatting rule that guides this? I often see such messages, especially in assertions, where the string is always on its own line. In these special constructs, a trailing string is expected and is a message back to the user.

Reply:
In assertions the string is on its own line, but there is no label associated with it. I believe we prefer to not unlink the label from the value it is applied to, in general.